src/styles/components/_dialog.scss

```diff
@@ -26,3 +26,7 @@
   margin-bottom: 0.5em;
   padding-bottom: 0.5em;
 }
+
+.dialog button {
```

Comment:
Try to follow the [BEM naming convention](http://getbem.com/naming/). (Same below)